plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -162,4 +164,54 @@ public static function get_automated_support_countries() {
 
 		return $tax_supported_countries;
 	}
+
+	/**
+	 * Get an array of countries that support Stripe tax.
+	 *
+	 * @return array
+	 */
+	public static function get_stripe_tax_support_countries() {
```

Comment:
What do you think if we change the function to be private so that we can remove it when we don't promote the Stripe tax?